client/src/components/ResponsiveAppBar/ResponsiveAppBar.js

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Menu from '@mui/material/Menu';
+import MenuIcon from '@mui/icons-material/Menu';
+import Container from '@mui/material/Container';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import Tooltip from '@mui/material/Tooltip';
+import MenuItem from '@mui/material/MenuItem';
+import AdbIcon from '@mui/icons-material/Adb';
+
+import { AppBar, Box, Toolbar, IconButton, Typography, Menu, MenuIcon, Container, Avatar, Button, Tooltip, MenuItem, AdbIcon} from '@mui/material';
+
```

Comment:
If you look at the original imports, you'll notice the icon name is actually "Adb" from MUI, so we will rename the import to AdbIcon. Import all icons like so: 
- TODO: Import all icons in the line below, and rename them to their respective icon name

```js
import { Adb as AdbIcon } from '@mui/icons-material';
```